assets/js/reducers/index.js

```diff
@@ -1,16 +1,17 @@
-import { combineReducers }  from 'redux';
-import { routerReducer }    from 'react-router-redux';
-import session              from './session';
-import registration         from './registration';
-import categories           from './categories';
-import matches              from './matches';
-import users                from './users';
+import { combineReducers } from 'redux';
+
```

Comment:
blank line on purpose?

Reply:
yup, it's not reducer, lines from 3 to 8 contain reducers